Mathlib/Algebra/Order/WithTop/Untop0.lean

```diff
@@ -77,27 +84,71 @@ lemma untop₀_mul [DecidableEq α] [MulZeroClass α] (a b : WithTop α) :
 ## Simplifying Lemmas in cases where α is a OrderedAddCommGroup
 -/
 
+section OrderedAddCommGroup
+
+variable
+  [AddCommGroup α] [PartialOrder α]
```

Comment:
How about doing this
```suggestion
variable [AddCommGroup α] [PartialOrder α] {a b : WithTop α}
```
and then re-using a and b below?

Reply:
Immplemented as suggested.